tests/app_crm/test_service.py

```
@@ -0,0 +1,165 @@
+    processor = build_processor(gateway)
+
+    # Seed an existing row
+    seed_values = [
+        "Office",
+        "CRM Core XP Tower 2",
+        "Офис",
+        250,
+        180,
+        "2 этаж",
+        "Аренда",
+        "С отделкой",
```

Ruff (0.14.2)

String contains ambiguous `С` (CYRILLIC CAPITAL LETTER ES). Did you mean `C` (LATIN CAPITAL LETTER C)?

(RUF001)